lib/components/c_facets/Drag.js

```diff
@@ -96,7 +96,7 @@ function Drag$start() {
  * @private
  */
 function _addDragAttribute() {
-    if (this.el) this.el.setAttribute('draggable', true);
+    if (!this.config.off && this.el) this.el.setAttribute('draggable', true);
```

Comment:
Ah, I see, you wanted to change the context to the facet? In that case please use the milo `{context, subscriber}` approach instead on bind. But then if `this` is a facet, you won't be able to have `this.el` anymore, it should be `this.owner.el`, no?

Reply:
I'm going to say probably yes. I'll take a closer look and fix it tomorrow, it took me long enough to figure out that while my initial change somehow broke 4-7 tests randomly each run of the tests in the project, all the other tests were fine, but one broken test because of `config` not being on the context was enough to bust them all.